managed/ui/src/components/universes/UniverseForm/UniverseFormContainer.js

```diff
@@ -61,9 +61,14 @@ import { toast } from 'react-toastify';
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    submitConfigureUniverse: (values) => {
+    submitConfigureUniverse: (values, uuid = null) => {
       dispatch(configureUniverseTemplateLoading());
       return dispatch(configureUniverseTemplate(values)).then((response) => {
+        if(response.error && uuid) {
+          dispatch(fetchUniverseInfo(uuid)).then((response) => {
+            dispatch(fetchUniverseInfoResponse(response.payload));
```

Comment:
`submitConfigureUniverse()` expected to return a promise at `createUniverse()`
pls make sure it works as expected after this change

Reply:
Yes, I was able to create a universe with this change in place, But will do it once more.